vocabularies/UI.xml

```diff
@@ -1600,6 +1600,16 @@ This is in contrast to [DataFieldForIntentBasedNavigation](#DataFieldForIntentBa
         <Annotation Term="Core.Description" String="The contents of this property must not be propagated to the app-to-app navigation context" />
       </Term>
 
+      <Function Name="toTimestamp">
+        <Annotation Term="Core.Description" String="Representation of the given point in time as an `Edm.DateTimeOffset` whose [timezone fragment](https://www.w3.org/TR/xmlschema11-2/#nt-tzFrag) is `Z`" />
+        <Parameter Name="Date" Type="Edm.Date"/>
+        <Parameter Name="TimeOfDay" Type="Edm.TimeOfDay"/>
+        <Parameter Name="Timezone" Type="Edm.String">
+          <Annotation Term="Core.Description" String="Time zone according to the [IANA](https://www.iana.org/time-zones) standard" />
+          <Annotation Term="Core.LongDescription" String="If absent, the time zone of the UI (Fiori Launchpad) is implied" />
+        </Parameter>
+        <ReturnType Type="Edm.DateTimeOffset"/>
+      </Function>
```

Comment:
This defines a server-side function. 

We should discuss in the OData TC whether this is also the way to go for defining client-side functions, and how to mark these functions as "client-side".

Could you please open a TC jira issue?

Reply:
https://issues.oasis-open.org/browse/ODATA-1428